blueprint/api.apib

```diff
@@ -1287,6 +1287,66 @@ A Stripe card belongs to a user.
 
     + Attributes (Record Not Found Response)
 
+# Group Stripe Customers
+
+This resource identifies a customer created on the [Stripe API](https://stripe.com/docs/api#customers).
+
+A Stripe customer can have many subscriptions.
+
+## Stripe Customers [/stripe-customers]
+
+### List all Stripe customers [GET]
+
```

Comment:
I would maybe make this a filter rather than list. See #396 for further discussion around this.